clients/python/girder_client/__init__.py

```diff
@@ -484,10 +489,10 @@ def sendRestRequest(self, method, path, parameters=None,
                 return result.json()
             else:
                 return result
-        # TODO handle 300-level status (follow redirect?)
```

Comment:
Is this comment being removed because 3** status codes never occur, or because they're already being handled somehow?